load.php

```diff
@@ -168,6 +186,38 @@ function perflab_render_generator() {
 }
 add_action( 'wp_head', 'perflab_render_generator' );
 
+/**
+ * Don't load the performance modules if the core version of the module is available.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $module The name of the module.
+ * @return bool Whether to load module or not.
```

Comment:
We call this "module slug" (or sometimes "module directory") elsewhere.
```suggestion
 * @param string $module Slug of the module.
 * @return bool Whether the module can be loaded or not.
```

Reply:
The docblock is updated. Thanks!